Hestia/app/src/main/java/hestia/UI/dialogs/AddDeviceDialog.java

```diff
@@ -50,7 +50,7 @@ public void setFragmentManager(FragmentManager fragmentManager) {
 
     @Override
     String buildTitle() {
-        return "new Add device";
+        return getContext().getString(R.string.addDeviceTitle);
```

Comment:
This quite a strange name for a dialog. I believe that simply "Add device" should do.